make/CMakeLists.txt

```diff
@@ -279,7 +279,7 @@ set (CORE_SOURCE
     ${CORE_DIR}/d-print.c
     ${CORE_DIR}/d-stack.c
     ${CORE_DIR}/d-trace.c
-    ${CORE_DIR}/e-func-symbols.c
+    ${CORE_DIR}/tmp-symbols.c
     ${CORE_DIR}/f-blocks.c
```

Comment:
This change is needed for this commit (correct source file directory, and add it to the list of generated files):
```
diff --git a/make/CMakeLists.txt b/make/CMakeLists.txt
index 74fabbd..366b391 100644
--- a/make/CMakeLists.txt
+++ b/make/CMakeLists.txt
@@ -279,7 +279,7 @@ set (CORE_SOURCE
     ${CORE_DIR}/d-print.c
     ${CORE_DIR}/d-stack.c
     ${CORE_DIR}/d-trace.c
-    ${CORE_DIR}/tmp-symbols.c
+    ${CORE_GENERATED_DIR}/tmp-symbols.c
     ${CORE_DIR}/f-blocks.c
     ${CORE_DIR}/f-deci.c
     ${CORE_DIR}/f-dtoa.c
@@ -421,6 +421,7 @@ set(HEADER_OUTPUT
     ${TOP_GENERATED_SRC_DIR}/include/tmp-funcs.h
     ${TOP_GENERATED_SRC_DIR}/include/tmp-funcargs.h
     ${TOP_GENERATED_SRC_DIR}/include/tmp-strings.h
+    ${CORE_GENERATED_DIR}/tmp-symbols.c
     )
```